tests/TestCase/Command/WorkerCommandTest.php

```diff
@@ -85,6 +85,24 @@ public function testQueueProcessesWithListener()
         $this->assertEmpty($this->getActualOutput());
     }
 
+    /**
+     * Test that queue will run by setting the processor name
+     *
+     * @runInSeparateProcess
+     */
+    public function testQueueProcessesWithProcessor()
```

Comment:
This test passes even without any changes, so it's not very useful. That said I don't see an easy way to check if the processor name option is actually used.

Reply:
It was a useless test, so I deleted it.
I'm sorry. I don't know a good way to check it either.

I was able to confirm that it worked all the way through the E2E test.